src/assets.rs

```diff
@@ -111,6 +111,13 @@ impl HighlightingAssets {
             );
         }
 
+        // Disabled for now. Prepration for improving startup speed.
+        if false {
+            // To trigger this code, run:
+            //   cargo run -- cache --build --source assets --blank --target /tmp
+            crate::syntax_analysis::print_syntax_dependencies(&syntax_set_builder);
+        }
```

Comment:
Do we want to merge this as-is? (with the `if false`?)

Other ideas:
- hide behind a cargo-feature (so we can trigger it with something like `cargo run --features print_syntax_dependencies -- cache --build …`)
- enable it with a environment variable `if std::env::var("BAT_PRINT_SYNTAX_DEPENDENCIES").is_ok() { …`. Contrary to the other two options, this would always include the necessary code in the binary. The env-var check wouldn't hurt at this place though.

Reply:
Thanks for the detailed code review!

I considered adding a cargo-feature for it, but since those are practically part of the public API, and since the if false is a temporary thing, I opted out of it.

I do however really like the idea of triggering the code with an env var. Will fix!